src/oauth2.h

```diff
@@ -34,15 +34,15 @@ constexpr const char kDefaultTokenEndpoint[] =
 class OAuth2 {
  public:
   OAuth2(const Environment& environment)
-    : OAuth2(environment, ExpirationImpl<std::chrono::system_clock>::New()) {}
+      : OAuth2(environment, ExpirationImpl<std::chrono::system_clock>::New()) {}
```

Comment:
Why did we change the indentation here? Should we just revert this file altogether?